api/rest/mantisbt_openapi.yaml

```diff
@@ -0,0 +1,1733 @@
+openapi: 3.0.0
+info:
+  title: Mantis Bug Tracker REST API
+  version: 2.27.0
+  license:
+    name: GPL-2.0
```

Comment:
*name* key's value should be `GNU General Public License v2.0 or later`.

SPDX-License expression should be specified under *identifier* key (then the _url_ key should be removed). Correct value for us is `GPL-2.0-or-later`.